resolve.js

```diff
@@ -13,18 +13,22 @@ const command = async (file, cmd) => {
     config.init(cmd);
     const jsonSchema = config.get('jsonSchema');
     const output = config.get('resolve:output');
-    const verbose = config.get('quiet') ? -1 : config.get('verbose');
+    const verbose = config.get('quiet') ? 0 : (config.get('verbose') ? 2 : 1);
 
     const spec = await loader.readOrError(file, buildLoaderOptions(jsonSchema, verbose));
     const content = yaml.safeDump(spec, { lineWidth: -1 });
 
     if (output) {
-        fs.writeFile(output, content, 'utf8', () => {
+        fs.writeFile(output, content, 'utf8', err => {
```

Comment:
Also fixed an issue where writing to a folder that didnt exist would fail silently.